src/constants/tokens.ts

```diff
@@ -353,25 +360,18 @@ function getCeloNativeCurrency(chainId: number) {
   }
 }
 
-export function isMatic(chainId: number): chainId is ChainId.POLYGON | ChainId.POLYGON_MUMBAI {
+export function isPolygon(chainId: number): chainId is ChainId.POLYGON | ChainId.POLYGON_MUMBAI {
```

Comment:
pedantically (after the rename of 2021) polygon is the chain & Matic is the token